source/common/tracing/zipkin/tracer.h

```diff
@@ -44,23 +44,33 @@ typedef std::unique_ptr<Reporter> ReporterPtr;
 class Tracer : public TracerInterface {
 public:
   /**
+   * Move constructor.
```

Comment:
Like my other comment, I would avoid using the move constructor here. You aren't actually moving most things, and it makes the code harder to follow. I would remove the move constructor, and refactor the code so that if you have to move a tracer, you should actually be moving a unique_ptr to a tracer.